lib/Config.ts

```diff
@@ -0,0 +1,92 @@
+import os from 'os';
+import path from 'path';
+import fs from 'fs';
+import toml from 'toml';
+import ini from 'ini';
+import { deepMerge } from './Utils';
+import { RpcConfig } from './rpc/RpcClient';
+
+class Config {
+  public config: string;
+  public btcdconfig: string;
+  public logfile: string;
+  public loglevel: string;
+  public rpc: RpcConfig;
+  private btcdDir: string;
+
+  constructor() {
+    const platform = os.platform();
+
+    switch (platform){
+      // TODO: Check if the btcd.conf file directory is proper for linux and mac
```

Comment:
For Linux it is `process.env.HOME/.btcd` and Mac is `process.env.HOME/Library/Application Support/Btcd` (but I am not sure).